pkgs/development/interpreters/cling/default.nix

```diff
@@ -73,10 +73,10 @@ let
   flags = [
     "-nostdinc"
     "-nostdinc++"
-    "-I" "${gcc-unwrapped}/include/c++/${gcc-unwrapped.version}"
```

Comment:
Nice touch. Then can drop gcc-unwrapped from the arguments in the top of this file.

Reply:
Done